src/app/pages/payment/pin-payment.component.ts

```diff
@@ -3,7 +3,7 @@ import { Component, OnInit } from '@angular/core';
 import { FormGroup, Validators, FormControl } from '@angular/forms';
 import { PaymentService } from 'src/app/services/payment/payment-service';
 import { NgxSpinnerService } from 'ngx-spinner';
-import { Router } from '@angular/router';
+import { Router, ActivatedRoute } from '@angular/router';
 import { ToastrService } from 'ngx-toastr';
 import { Location } from '@angular/common';
 import { Title } from '@angular/platform-browser';
```

Comment:
Great work @CryceTruly , lets try to follow convention here, the top imports should be angular core imports followed by third party imported libraries

Reply:
hey @Genza999, i have worked on this.